src/odemis/driver/test/technolution_test.py

```diff
@@ -103,6 +109,10 @@ class TestAcquisitionServer(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
+        if not technolution_available:
+            raise unittest.SkipTest(f"Skipping the technolution tests, correct libraries to perform the tests are not"
+                                f" available.")
```

Comment:
can you also add this to the other test classes? This way all test cases are skipped if technolution_asm is not available